track/templates/en/layout-en.html

```diff
@@ -55,8 +55,13 @@ <h2 class="py-6 text-base sm:text-lg hidden md:block"><a class="text-grey-darkes
 		                    <a class="text-https-blue hover:text-black no-underline hover:underline" href="/en/organizations/">Check compliance</a>
 		                </li>
 		                <li class="mr-8 py-6 hidden xl:flex">
-		                    <a class="text-https-blue hover:text-black no-underline hover:underline" href="/en/guidance/">Read guidance</a>
+		                    <a class="text-https-blue hover:text-black no-underline hover:underline" href="/en/guidance/">Project guidance</a>
+						</li>
```

Comment:
Can you configure your editor to use spaces instead of tabs?

The source is going to get pretty messy if we start to mix formatting styles

Reply:
oh ... ok will double check that.